### PR TITLE
Update TOC to include SLA

### PR DESCRIPTION
**Pull request by jamesw-msft**
Update TOC to include SLA

### CODE REVIEW

**Review by msewaweru — CHANGES_REQUESTED** [review]
@jamesw-msft - See my comment below for your action.

**Review by msewaweru — APPROVED** [review]
@jamesw-msft -LGTM! Thanks.

File: api-reference/beta/toc.yml [hunk 1]
```diff
@@ -15348,6 +15348,16 @@ items:
               href: api/reportroot-list-applicationsignindetailedsummary.md
             - name: Get sign-in details
               href: api/applicationsignindetailedsummary-get.md
+        - name: Azure AD SLA attainment
+          items:
+          - name: Azure AD Authentication
+            href: resources/azureadauthentication.md
+          - name: Service Level Agreement
+            href: resources/servicelevelagreementroot.md
+          - name: Service Level Agreement attainment
```
Comment (Lauragra) [inline api-reference/beta/toc.yml:15357]: Per our guidelines, we don't surface complex types in the TOC. I think here the TOC structure can simply be:

Azure AD authentication
    Get

Also note that per our guidelines, we use sentence case in the TOC, so only the first letter is capitalized except for proper nouns.

Comment (jamesw-msft) [reply api-reference/beta/toc.yml]: Removed the complex type and fixed the casing per our chat offline.

**Pull request merged**